content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -452,20 +452,30 @@ for more information.
 
 ## pod-security.kubernetes.io/exempt
 
+
 Example: `pod-security.kubernetes.io/exempt: namespace`
 
-Used on: Event
+Used on: `audit.k8s.io/Event`
 
 Value **must** be one of `user`, `namespace`, or `runtimeClass` which correspond to
 [Pod Security Exemption](/docs/concepts/security/pod-security-admission/#exemptions)
 dimensions. This annotation indicates on which dimension was based the exemption
 from the PodSecurity enforcement.
 
+{{< caution >}}
```

Comment:
Use a note, not a caution. `{{< caution >}}` is for advice you should follow or you might make a mistake that's bad for youc clisyer.

Reply:
Oops!